addons/binding/org.openhab.binding.minecraft/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="minecraft"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <bridge-type id="server">
+        <label>Server</label>
+        <description>Server from which data is being fettched</description>
+
+        <channels>
+            <channel id="name" typeId="name"/>
+            <channel id="online" typeId="online"/>
+            <channel id="bukkitVersion" typeId="bukkitVersion"/>
+            <channel id="version" typeId="version"/>
+            <channel id="players" typeId="players"/>
+            <channel id="maxPlayers" typeId="maxPlayers"/>
+        </channels>
+
+	    <config-description>
+           <parameter name="hostname" type="text" required="true">
+               <label>Hostname or IP</label>
+               <description>Hostname or IP of the device</description>
+           </parameter>
+
+           <parameter name="port" type="integer">
+               <label>Port</label>
+               <description>The port on which the server can be accessed</description>
+               <default>10692</default>
+           </parameter>
+       </config-description>
+    </bridge-type>
+    
+    <thing-type id="player">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="server"/>
+        </supported-bridge-type-refs>
+        
+        <label>Player</label>
+        <description>A connected player on minecraft server</description>
+
+        <channels>
+            <channel id="playerName" typeId="playerName"/>
+            <channel id="playerOnline" typeId="playerOnline"/>
+            <channel id="playerLevel" typeId="playerLevel"/>
+            <channel id="playerTotalExperience" typeId="playerTotalExperience"/>
+            <channel id="playerExperiencePercentage" typeId="playerExperiencePercentage"/>
+            <channel id="playerHealth" typeId="playerHealth"/>
+            <channel id="playerWalkSpeed" typeId="playerWalkSpeed"/>
+            <channel id="playerLocationX" typeId="playerLocationX"/>
```

Comment:
Instead of x,y,z, did you consider to have a single location channel, which can be linked to a "Location" item? I would prefer this.